modules/administration-guide/partials/proc_configuring-minikube-github-authentication.adoc

```diff
@@ -0,0 +1,59 @@
+[id="configuring-minikube-github-authentication_{context}"]
+= Configuring Minikube with GitHub Authentication
```

Comment:
Is the procedure only for Minikube, excluding other Kubernetes distributions? Or for Kubernetes excluding OpenShift?

Reply:
minikube only